pipeline.yml

```diff
@@ -14,18 +14,27 @@ nodejs_version: "10"
 build:
   customized_builder_image:
     registry: artifactory
-    url: artifactory.dev.adskengineer.net/docker-local-v2/docker-local-v2/dynamo/dynamolibrariejs:latest
+    url: artifactory.dev.adskengineer.net/docker-local-v2/dynamo/dynamolibrariejs:dotnetcore2
   before_scripts:
     - "npm install"
   scripts:
-    - "npm run build"  
+    - "npm run build"
 
 ci_test:
   customized_builder_image:
     registry: artifactory
-    url: artifactory.dev.adskengineer.net/docker-local-v2/docker-local-v2/dynamo/dynamolibrariejs:latest
+    url: artifactory.dev.adskengineer.net/docker-local-v2/dynamo/dynamolibrariejs:dotnetcore2
   scripts:
     - "npm run test"
     - "npm run utiltest"
-  
-  test_report_dir: TestResults
\ No newline at end of file
+  test_report_dir: TestResults
+
+deployment:
+-
+  type: customized
+  customized_builder_image:
+    registry: artifactory
+    url: artifactory.dev.adskengineer.net/docker-local-v2/dynamo/dynamolibrariejs:dotnetcore2
+  scripts:
+    - "dotnet pack -c Release -o nuget --version-suffix $BUILD_NUMBER"
+    - "dotnet nuget push \"$(find nuget -type f -name *.nupkg)\" -s https://art-bobcat.autodesk.com/artifactory/api/nuget/team-dynamo-nuget -k $API_KEY"
```

Comment:
did you verify with @smangarole that we want to publish this to internal nuget?
I cannot recall 100% but I thought we discussed publishing to nuget as this is an open source repo that Dynamo will know rely on.

Reply:
@mjkkirschner  Please take a look on that, we always talk about the artifactory as the ADSK aritifactory: https://jira.autodesk.com/browse/DYN-2117